fastlane/actions/podspec_constant_versioning.rb

```diff
@@ -0,0 +1,78 @@
+module Fastlane
+  module Actions
+    class PodspecConstantVersioningAction < Action
+      def self.run(params)
+        podspec = params[:podspec]
+        constant = params[:constant]
+        value = params[:value]
+        
+        podspec_contents = File.read(podspec)
+
+        # TODO: improve RegEx
+        if !podspec_contents.match(/(?<=#{constant})(\s*=\s*["'].*)/)
+          UI.error("#{constant} not present or doesn't have an explicit value in #{podspec}")
+          return
+        end
+
+        new_value = " = '#{value}'"
+        
+        podspec_new_contents = podspec_contents.gsub(/(?<=#{constant})(\s*=\s*["'].*)/, new_value)
```

Comment:
I think this is the same regex as used in the match? If so, definitely want this in a separate variable